testsuite/integration-arquillian/servers/pom.xml

```diff
@@ -38,14 +38,13 @@
         <eap.version>7.0.0.ER6-redhat-1</eap.version>
         <eap6.version>7.5.6.Final-redhat-2</eap6.version>
         <jboss.as.version>7.1.1.Final</jboss.as.version>
-        <eap6.version>7.5.6.Final-redhat-2</eap6.version>
         <tomcat7.version>7.0.68</tomcat7.version>
         <tomcat8.version>8.0.32</tomcat8.version>
         <tomcat9.version>9.0.0.M3</tomcat9.version>
         <karaf3.version>3.0.3</karaf3.version>
         <fuse61.version>6.1.0.redhat-379</fuse61.version>
         <!--<fuse62.version>6.2.0.redhat-133</fuse62.version>-->
-        <fuse62.version>6.2.1.redhat-084</fuse62.version>
+        <fuse62.version>6.2.1.redhat-103</fuse62.version>
```

Comment:
`6.2.1.redhat-103` is not a released version of the product, so it's not supposed to be permanently available on public maven servers. You should use `6.2.1.redhat-084` or `6.2.1.redhat-084` or wait for the tag for rollup patch 2 to be confirmed.